user-guide/Standard_Apps/Vecima Remote PHY Monitor/Remote_PHY_Monitoring_Installation.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Remote_PHY_Monitoring_Installation
+---
+# Installing Remote PHY Monitoring
+
+1. Make sure the following **prerequisites** are met:
+
+   - Your DataMiner System uses DataMiner 10.3.0 or higher.
+
+     > [!IMPORTANT]
+     > If you are using a version older than 1.0.4.0, upgrade the DataMiner web apps version to DataMiner 10.3.11 or higher.
```

Comment:
Is "1.0.4" the version of the solution here? This seems odd in a procedure for first-time installation. Would it make sense to instead add as a requirement that the web apps must be upgraded to 10.3.11 or higher in all cases?

Reply:
Yes, we can write that, but in February version 1.0.4.0 will be released and in that version we will no longer have to update the web applications.

Reply:
I'd suggest to add this as a requirement as follows then:

```
   - If version 1.0.4.0 or older of the solution is used, the DataMiner web apps must be upgraded to DataMiner 10.3.11 or higher.
```